src/nifreeze/utils/iterators.py

```diff
@@ -33,14 +33,17 @@ def linear_iterator(size: int | None = None, **kwargs) -> Iterator[int]:
 
     Parameters
     ----------
-    size : :obj:`int`
-        Number of volumes in the dataset
-        (for instance, the number of orientations in a DWI)
+    size : :obj:`int` or ``None`` (optional)
+        Number of volumes in the dataset.
+    bvalues : :obj:`list`
```

Comment:
I assume this is contained in the kwargs: not sure what the standard way to document this is. In any case, it should be marked as optional or something else, as it comes after an optional argument.

Reply:
```suggestion
    bvalues : :obj:`list`, optional (keyword argument)
```